onnxruntime/core/providers/cuda/shared_inc/fpgeneric.h

```diff
@@ -34,11 +34,15 @@ inline cublasStatus_t cublasGemmHelper(cublasHandle_t handle, cublasOperation_t
                                        int m, int n, int k, const half* alpha, const half* A, int lda,
                                        const half* B, int ldb, const half* beta, half* C, int ldc,
                                        const cudaDeviceProp& prop) {
+#ifndef ENABLE_TRAINING
```

Comment:
>ENABLE_TRAINING [](start = 8, length = 15)

would it cause difference in inference-only session when training is enabled?

Reply:
@KeDengMS inference-only session have this before. We disabled it when merge training code in master. I think  we just make the state back to their original state. Does this answer you question?

Reply:
My question is about the inference-only session in a build with ENABLE_TRAINING, vs. a inference-only session in a build without ENABLE_TRAINING. If it's just performance different, I think it's OK.